assets/templates/partials/banners/beta.tmpl

```diff
@@ -1,3 +1,21 @@
+{{ if eq .Type "search" }}
+<div class="ons-phase-banner">
+  <div class="ons-container">
+    <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap">
+      <div class="ons-grid__col ons-col-auto ons-u-flex-no-grow ons-u-flex-no-shrink">
+        <h3 class="ons-phase-banner__badge">{{ localise "Beta" .Language 1 }}</h3>
+      </div>
+        <span class="ons-u-fs-s ons-u-m-no">{{ localise "BetaSearchFeedback" .Language 1 | safeHTML }}</span>
+            <form action="/search" method="get">
+                <input type="hidden" id="q" name="q" value="{{ .Data.Query }}">
+                <input type="hidden" id="exit-new-search" name="exit-new-search" value="true">
+                <button type="submit" role="link" class="default-line-height decoration-line-thickness ons-btn ons-btn__btn--link margin-left--half font-size--14" name="previous version" data-gtm-search-exit="true">You can still use the previous version</button>
```

Comment:
`margin-left--half font-size--14` are Sixteens utility classes, can you use the equivalent from the [design-system](https://github.com/ONSdigital/design-system)? 
e.g. `ons-u-ml-s` for margin-left 
e.g. `ons-u-fs-s` for font-size of 14px